newrelic-agent/src/main/java/com/newrelic/agent/TransactionService.java

```diff
@@ -71,12 +63,7 @@ public TransactionService(int numMaintenanceThreads, long initialDelay, long del
 
         // run as daemon to not prevent shutdown of application
         scheduler = Executors.newScheduledThreadPool(numMaintenanceThreads, new DefaultThreadFactory(TRANSACTION_SERVICE_PROCESSOR_THREAD_NAME, true));
-        scheduler.scheduleWithFixedDelay(new Runnable() {
-            @Override
-            public void run() {
-                processQueue();
-            }
-        }, initialDelayMilli, delayMilli, TimeUnit.MILLISECONDS);
+        scheduler.scheduleWithFixedDelay(() -> TransactionService.this.processQueue(), initialDelayMilli, delayMilli, TimeUnit.MILLISECONDS);
```

Comment:
OMG is this a lambda in the Java agent code base!!! It's happening!!!

Reply:
Yes, that was my little test.  That caused a bug that took a while to resolve. The error suggested the android guava dependency might be the issue.   I ended up deleting my gradle cache and starting from scratch again.